shared-files/_glossary.md

```diff
@@ -0,0 +1,324 @@
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Rancher server, Upstream cluster</i>
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Managed cluster</i>
+    </p>
+</dl>
+
+## M
+
+<dl>
+    <dt>
+        Managed cluster
+    </dt>
+    <dd>
+        A Kubernetes cluster that is managed by a Rancher server. Managed clusters can run on a variety of infrastructure and can be located anywhere, including in the cloud, on-premises, or in air-gapped environments. 
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Downstream cluster</i>
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Rancher server</i>
+    </p>
+</dl>
+
+## R
+
+<dl>
+    <dt>
+        Rancher Enterprise
+    </dt>
+    <dd>
+        An alternative name for <i>Rancher Prime</i>. 
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Rancher Prime</i>
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Community</i>
+    </p>
+    <dt>
+        Rancher Kubernetes Engine (RKE)
+    </dt>
+    <dd>
+        <ol>
+            <li> 
+                A CNCF-certified Kubernetes distribution that runs entirely within Docker containers. There are two main versions of RKE available. The 1.x software line is sometimes called <i>RKE1</i>. The next-generation line is called <i>RKE2</i> or <i>RKE Government</i>.
+            </li>
+            <li>
+                Shorthand for <i>RKE1</i>, the 1.x software line of RKE.
+            </li>
+        </ol>
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>K3s, RKE1, RKE2, RKE Government</i>
+    </p>
+    <dt>
+       Rancher Prime 
+    </dt>
+    <dd>
+        Rancher Prime is a new edition of the commercial, enterprise offering built on the the same source code. Installation assets are hosted on a trusted registry owned and managed by Rancher with additional value coming in from security assurances, extended lifecycles, access to focused architectures and Kubernetes advisories.
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Rancher Enterprise</i>
+    </p>
+    <dt>
+        Rancher server
+    </dt>
+    <dd>
+        Rancher Server is a Kubernetes management tool to deploy and run clusters anywhere and on any provider. Rancher includes all the software and downstream cluster components used to manage the entire Rancher deployment.
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Local cluster, Upstream cluster</i>
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Managed cluster</i>
+    </p>
+    <dt>
+        RKE1
+    </dt>
+    <dd>
+        An alternative name for the 1.x software line of Rancher Kubernetes Engine (RKE). Sometimes called <i>RKE</i> for short. RKE1 is a certified Kubernetes distribution and CLI/library which creates and manages a Kubernetes cluster.
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Rancher Kubernetes Engine, RKE2</i>
+    </p>    
+    <dt>
+        RKE2
+    </dt>
+    <p>
+        <b>Versions:</b> Rancher v2.6.0 and later
```

Comment:
`Rancher v2.6.0 and later` this is only accurate for provisioning. In v2.5 it's possible to register RKE2 clusters. Not sure how we should update this without causing the version field to be super inconsistent though.

Reply:
Just applied the parentheses idea